pkg/backend/filestate/backend.go

```diff
@@ -401,7 +422,7 @@ func (b *localBackend) apply(ctx context.Context, kind apitype.UpdateKind, stack
 		fmt.Printf(
 			op.Opts.Display.Color.Colorize(
 				colors.SpecHeadline+"Permalink: "+
-					colors.Underline+colors.BrightBlue+"file://%s"+colors.Reset+"\n"), stack.(*localStack).Path())
+					colors.Underline+colors.BrightBlue+"%s%s"+colors.Reset+"\n"), b.urlScheme(), stack.(*localStack).Path())
```

Comment:
What kind of a link is this? Is the user expected to be able to click on it? The gocloud.dev URLs work for gocloud.dev APIs, but not in your browser. If you need the latter, you can use `Bucket.SignedURL`.

Reply:
And what does `Path()` return?